templates/helm/orchestration-workflows/templates/clinvar.yaml

```diff
@@ -18,8 +19,7 @@ spec:
       k8sName: argo-runner
       googleName: clinvar-argo-runner@broad-dsp-monster-{{ .Values.env }}.iam.gserviceaccount.com
     cron:
-      # 8:00am every day for now, as we test.
-      schedule: "0 8 * * *"
+      schedule: {{ .Values.clinvar.schedule }}
```

Comment:
This is now being passed in as an argument on the hack/apply-orchestration-workflows chart, right?

Reply:
Ah I see it, line 19. Nvm!